package.json

```diff
@@ -5,13 +5,13 @@
   "private": true,
   "license": "MPLv2",
   "scripts": {
-    "start:a": "npm run build && web-ext run -s dist/extension-a-$npm_package_version",
-    "start:b": "npm run build && web-ext run -s dist/extension-b-$npm_package_version",
-    "start:c": "npm run build && web-ext run -s dist/extension-c-$npm_package_version",
+    "start:a": "npm-run-all build && web-ext run -s dist/extension-a-$npm_package_version",
+    "start:b": "npm-run-all build && web-ext run -s dist/extension-b-$npm_package_version",
+    "start:c": "npm-run-all build && web-ext run -s dist/extension-c-$npm_package_version",
```

Comment:
Sorry, but can you also use the long form of the flags here too? It really helps with future readability for people that aren't familiar with the command.